src/redteam/index.ts

```diff
@@ -273,20 +277,15 @@ export async function synthesize({
     if (plugin) {
       updateProgress();
       logger.debug(`Generating ${key} tests`);
-      const pluginTests = await action(
-        redteamProvider,
-        purpose,
-        injectVar,
-        plugin.numTests,
-        prompts,
-      );
+      const pluginTests = await action(redteamProvider, purpose, injectVar, plugin.numTests);
       testCases.push(
         ...pluginTests.map((t) => ({
           ...t,
           metadata: {
             ...(t.metadata || {}),
             pluginId: key,
             purpose,
+            ...(key === 'imitation' && entities.length > 0 ? { entities } : {}),
```

Comment:
nitnit: maybe make  `INCLUDE_ENTITY_METADATA = ['imitation']` or similar, so that it's easier to find in the future when we add entities to `competitors` etc